Assets/HoloToolkit/Boundary/Scripts/BoundaryManager.cs

```diff
@@ -191,10 +191,6 @@ public void CalculateBoundaryVolume()
                     }
                 }
             }
-            else
```

Comment:
Looks like this is removing something that was added in the Dev branch.

Reply:
Do we really need a log that this always returns false when it's stated in the comments above?

Reply:
This is kinda strange, I did a straight merge from the master branch, without any edits.

Reply:
I think this was added by @NeerajW as a way to validate when that API becomes fixed on Unity's end.

Reply:
Idk, having optional debug logs like this is probably useful.  IMO, Production apps wouldn't want this kind of thing spamming the console.

Reply:
Not to mention the performance impact debug statements have in general.